tests/csm/rest/test_query_deployment.py

```diff
@@ -0,0 +1,113 @@
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""Tests Query Deployment scenarios using REST API
+"""
+
+import time
+import logging
+
+import pytest
+
+from commons import configmanager, cortxlogging
+from commons.constants import K8S_SCRIPTS_PATH, K8S_PRE_DISK
+from libs.csm.csm_interface import csm_api_factory
+from libs.ha.ha_common_libs_k8s import HAK8s
+from libs.prov.prov_k8s_cortx_deploy import ProvDeployK8sCortxLib
+
+
+class TestQueryDeployment():
+    """Query Deployment Testsuites"""
+
+    @classmethod
+    def setup_class(cls):
+        """Setup class"""
+        cls.log = logging.getLogger(__name__)
+        cls.ha_obj = HAK8s()
+        cls.deploy_lc_obj = ProvDeployK8sCortxLib()
+        cls.csm_obj = csm_api_factory("rest")
+        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.update_seconds = cls.csm_conf["update_seconds"]
+
+    def setup_method(self):
+        """
+        Setup method
+        """
+        self.log.info("Prerequisite: Deploy cortx cluster")
+        self.log.info("Cleanup: Destroying the cluster ")
+        resp = self.deploy_lc_obj.destroy_setup(self.csm_obj.master, self.csm_obj.worker_list,
+                                                K8S_SCRIPTS_PATH)
+        assert resp[0], resp[1]
+        self.log.info("Cleanup: Cluster destroyed successfully")
+
+        self.log.info("Cleanup: Setting prerequisite")
+        self.deploy_lc_obj.execute_prereq_cortx(self.csm_obj.master,
+                                                K8S_SCRIPTS_PATH,
+                                                K8S_PRE_DISK)
+
+        for node in self.csm_obj.worker_list:
+            self.deploy_lc_obj.execute_prereq_cortx(node, K8S_SCRIPTS_PATH,
+                                                    K8S_PRE_DISK)
+        self.log.info("Cleanup: Prerequisite set successfully")
+
+        self.log.info("Cleanup: Deploying the Cluster")
+        resp_cls = self.deploy_lc_obj.deploy_cluster(self.csm_obj.master,
+                                                     K8S_SCRIPTS_PATH)
+        assert resp_cls[0], resp_cls[1]
+        self.log.info("Cleanup: Cluster deployment successfully")
+
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+        time.sleep(self.update_seconds)
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+
+        self.log.info("Cleanup: Check cluster status")
+        resp = self.ha_obj.poll_cluster_status(self.csm_obj.master)
+        assert resp[0], resp[1]
+        self.log.info("Cleanup: Cluster status checked successfully")
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-45672')
+    def test_45675(self):
```

Comment:
why this test id is mismatched with above marker?

Reply:
Correct one 45675

Reply:
corrected